pkgs/top-level/all-packages.nix

```diff
@@ -2273,12 +2273,10 @@ with pkgs;
   elk6Version = "6.3.2";
 
   elasticsearch = callPackage ../servers/search/elasticsearch { };
-  elasticsearch2 = callPackage ../servers/search/elasticsearch/2.x.nix { };
-  elasticsearch5 = callPackage ../servers/search/elasticsearch/5.x.nix { };
-  elasticsearch6 = callPackage ../servers/search/elasticsearch/6.x.nix { };
```

Comment:
We should also add an entry to the [release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-1809.xml#L95) warning that `elasticsearch` has been upgraded to version 6.

We should probably also mention that `elasticsearch` now has an unfree license because it contains [X-pack](https://www.elastic.co/products/x-pack/open) but that you can use the free `elasticsearch-oss` instead.